docs/transparent-validation/muxpi/basic-validation.md

```diff
@@ -17,26 +18,33 @@
 ## NanoPi setup
 
 1. Prepare microSD card (minimum 8GB) for NanoPi NEO operating system.
-1. Download and extract image - [link](https://cloud.3mdeb.com/index.php/s/n42rLcAQ5cWcxgW/download)
-1. Flash image into microSD card ([Etcher](https://www.balena.io/etcher/) tool could be helpful)
+1. Download and extract image -
+[link](https://cloud.3mdeb.com/index.php/s/n42rLcAQ5cWcxgW/download)
+1. Flash image into microSD card ([Etcher](https://www.balena.io/etcher/) tool
+could be helpful)
 
 ## muxPi setup
 
+![](https://cloud.3mdeb.com/index.php/s/o3D5fwWbjscXDw9/preview)
+
 1. Attach NanoPi to muxPi connectors.
 1. Plug ETH cable to RJ45 NanoPi port.
 1. Plug microUSB<->USB cable to muxPi and your machine.
 1. Plug in power supply.
 1. Make sure that jumpers are correctly set according to image:
-![](../../images/muxPi-jumpers.png)
+![](https://cloud.3mdeb.com/index.php/s/EMpXf8c4JaDNTpH/preview)
+
+1. Connect to NanoPi using terminal
 
-1. Connect to NanoPi using terminal:
-`sudo minicom -D /dev/ttyUSB0 -o -b 115200`
+   sudo minicom -D /dev/ttyUSB0 -o -b 115200`
 
 1. Save ip address:
-`ifconfig`
+
+      ifconfig
```

Comment:
Code blocks should be added as follows:

```bash
<insert Your code here>
```
Please refer to another files in documentation.